pages/en/billing.mdx

```diff
@@ -2,75 +2,74 @@
+2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect".
 
-### Understanding Invoices in Subgraph Studio
+3. Click the dropdown next to the 'Add GRT' button at the center of the page. Select withdraw GRT. A side panel will appear.
 
-When using The Graph Network, dapps and users pay for query fees in GRT, the protocol's native work-utility token. Maintaining a positive balance for an account is the user's or dapp's responsibility.
+4. Enter the amount of GRT you would like to withdraw.
 
-Here are ways to be notified about and/or check a query fee balance:
+5. Click 'Withdraw GRT' to withdraw the GRT from your account. Sign the associated transaction in your wallet. This will cost gas. The GRT will be sent to your Arbitrum wallet.
 
-- You can set up a billing notification that will be sent to an email address [here](https://thegraph.com/studio/settings/).
+6. Once the transaction is confirmed, you'll see the GRT withdrawn from your account in your Arbitrum wallet within an hour.
 
-- You can query the billing subgraph API directly [here](https://thegraph.com/hosted-service/subgraph/graphprotocol/billing)
+### Adding GRT using a multisig wallet
 
-Also, you can learn how to (re)fill your query fee balance by watching [this video](https://www.loom.com/share/a81de6ef11d64c62872ea210c58c6af5).
+1. Go to the [Subgraph Studio Billing page](https://staging.thegraph.com/studio/billing/).
 
-> IMPORTANT: If there are unpaid invoices for more than two weeks, Indexers will stop serving queries for all API Keys associated with the account. Indexers will automatically start serving queries again once the billing balance is paid in full and GRT is added to the account for future query fees.
+2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect". If you're using [Gnosis-Safe](), you'll be able to connect your multisig as well as your signing wallet. Then, sign the associated transaction. This will not cost any gas.
 
-Lastly, learn how to prioritize price, speed, data freshness (among other factors) for your queries by watching [this video](https://www.loom.com/share/b5fc533e48584cb694017392c80c75e0). If you have any questions, please ask in [The Graph Discord](https://discord.gg/fjDWCTZE) and the community will be happy to help.
+3. Click the 'Add GRT' button at the center of the page. A side panel will appear.
 
-### How to Fill Your Account With a Multisig Wallet
+4. Enter the amount of GRT you would like to use and click the button to allow access to your GRT. This transaction will not require a gas fee. Ensure all signers have approved the transaction, and execute the transaction.
```

Comment:
It won't require a gas fee _at first_ but once all signers have approved, executing the transaction will. So I wonder if its best to leave out the part about it not requiring a gas fee. Thoughts?

Reply:
You're right, I'll fix that. I think it's best to say that it _will_ include a gas fee.